source/funkin/ui/story/StoryMenuState.hx

```diff
@@ -336,6 +336,24 @@ class StoryMenuState extends MusicBeatState
           changeDifficulty(0);
         }
 
+        #if !html5
+        if (FlxG.mouse.wheel != 0)
+        {
+          changeLevel(-Math.round(FlxG.mouse.wheel));
+        }
+        #else
+        if (FlxG.mouse.wheel < 0)
+        {
+          changeLevel(-Math.round(FlxG.mouse.wheel / 8));
+        }
+        else if (FlxG.mouse.wheel > 0)
+        {
+          changeLevel(-Math.round(FlxG.mouse.wheel / 8));
+        }
+        #end
+
+        // HTML and NON HTML builds mouse fix (JVN).
```

Comment:
its not cool to try to sneek urself in the code lmao

Reply:
yep